NEWS.md

```diff
@@ -33,8 +33,30 @@ The format is based on [Keep a Changelog]
 
 ## [unreleased]
 
+Note to package maintainers: The SSD titlebar window icon support requires
+libsfdo to be added as a dependency if your distribution want to statically
+link. If this is not wanted, add -Dicon=disabled to the `meson setup` command in
+the build script for the next release.
```

Comment:
`s/if your distribution want to statically link/or statically linked/`

Reply:
Done